src/styles/clock.scss

```diff
@@ -0,0 +1,142 @@
+.clock {
+  position: absolute;
+  height: 200px;
+  width: 200px;
+  background-color: #fff;
+  border-radius: 100%;
+  border: 10px solid #34626c;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  margin: auto;
+
+  &__top {
+    position: absolute;
+    width: 3px;
+    height: 8px;
+    background-color: #262626;
+    left: 0;
+    right: 0;
```

Comment:
Pay attention to your CSS properties order
https://9elements.com/css-rule-order/